packages/backend/src/tokens/authStatus.ts

```diff
@@ -158,13 +158,27 @@ const withDebugHeaders = <T extends RequestState>(requestState: T): T => {
   const headers = new Headers(requestState.headers || {});
 
   if (requestState.message) {
-    headers.set(constants.Headers.AuthMessage, requestState.message);
+    try {
+      headers.set(constants.Headers.AuthMessage, requestState.message);
+    } catch (e) {
+      // headers.set can throw if unicode strings are passed to it. In this case, simply do nothing
```

Comment:
@nikosdouvlis shouldn't we fall back to a specific value? or we don't really care about it?

Reply:
It's a very rare case that would otherwise cause the current request to fail - I don't think we need additional information in this case as the only way to reach this state is by using a forged/invalid token in the first place